domains/comi.is-local.org.json

```diff
@@ -0,0 +1,14 @@
+{
+    "description": "for my personal site ",
+    "domain": "is-local.org",
+    "subdomain": "comi",
+    "owner": {
+        "email": "comi@disroot.org"
+    },
+
+    "record": {
+        "CNAME": ["comi-xf.github.io/blog"]
```

Comment:
Your CNAME record can not have a `/` in it, so you will have to remove that to make the cname only be “comi-xf.github.io” otherwise it won’t work

Reply:
OK